runtime-modules/content/src/tests/nft/make_bid.rs

```diff
@@ -807,7 +808,7 @@ fn make_bid_ok_with_english_auction_completion_and_total_balance_slashed() {
         ));
 
         assert_eq!(
-            Balances::<Test>::free_balance(&SECOND_MEMBER_ACCOUNT_ID),
+            Balances::<Test>::usable_balance(&SECOND_MEMBER_ACCOUNT_ID),
```

Comment:
I think in this case it would actually be best to use:
```suggestion
            Balances::<Test>::total_balance(&SECOND_MEMBER_ACCOUNT_ID),
```
Because:
- It's a better way to make sure that the funds are actually slashed (not just reserved, locked etc.)
- It's inline with the test function name (make_bid_ok_with_english_auction_completion_and_**total_balance**_slashed)

The same applies to _make_bid_ok_with_open_auction_completion_and_total_balance_slashed_

Reply:
I would keep it as `usable_balance`. The main reason would be simplicity of the reasoning - 
 we use the same method most of the time. 

Parity's balances pallet is overcomplicated. After years with Substrate it's still unclear for me what method describes what balance without looking at the code. So, I'm hesitating to introduce `total_balance` without a mandatory case.